### ISSUE TITLE
[Polish] Accordion Block

### ISSUE THREAD

**Issue by stephaniefois** [vigetlabs/wordpress-site-starter#61]
- Styling
    - Design adherence (plus signs instead of arrows, plus signs in line and justified right)
    - Lists don’t have bullet points
- Functional
    - Include more formatting options in the toolbar (unordered list, bulleted list, media)

**Comment by nathan-schmidt-viget** [comment]
The styling part is done.

**Comment by bd-viget** [comment]
> Include more formatting options in the toolbar (unordered list, bulleted list, media)

@stephaniefois I'm not sure I understand what you mean by this. Can you elaborate a bit more?

**Comment by stephaniefois** [comment]
I guess I just expect that when I'm editing text I get a WYSIWYG toolbar instead of having to insert a block to get to some of those formatting options. 
<img width="401" alt="CleanShot 2024-05-21 at 10 27 49@2x" src="https://github.com/vigetlabs/wordpress-site-starter/assets/54903017/1f4490e7-42d4-4c6c-9c0e-533daa348741">

This may buck the system Gutenberg is trying to establish so I can be overruled on this one.

**Comment by bd-viget** [comment]
@stephaniefois Yeah, I think this should be marked as done as we're using `InnerBlocks` inside the Accordion Items instead of a WYSIWYG editor, but you can still add lists, bold, adjust text orientation, etc inside this element.
